src/common/ipc/ipc.ts

```diff
@@ -46,6 +48,7 @@ function getSubFrames(): ClusterFrameInfo[] {
 }
 
 export function broadcastMessage(channel: string, ...args: any[]) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Why this is needed? 🤔

Reply:
I don't know, I think a relic of a revert from master.